libgoal/libgoal.go

```diff
@@ -1049,18 +1052,18 @@ func MakeDryrunStateGenerated(client Client, txnOrStxn interface{}, other []tran
 						return
 					}
 					appParams = app.Params
+					accounts = append(accounts, appIdx.Address())
 				}
 				dr.Apps = append(dr.Apps, generatedV2.Application{
 					Id:     uint64(appIdx),
 					Params: appParams,
 				})
 			}
 
-			accounts := append(tx.Accounts, tx.Sender)
 			for _, acc := range accounts {
 				var info generatedV2.Account
 				if info, err = client.AccountInformationV2(acc.String()); err != nil {
-					return
+					// ignore error and continue
```

Comment:
Should this actually `continue` so that the empty account info isn't appended to `dr.Accounts`?